.github/workflows/ci.yml

```diff
@@ -312,3 +312,77 @@ jobs:
       with:
         name: release-artifacts
         path: release-artifacts
+  build-integration-tests-linux:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - run: |
+          set -ex
+          cd src/integration-tests
+          mkdir -p artifacts
+          cp integration-test.py artifacts/
+
+          mkdir -p artifacts/linux-libfuzzer
+          (cd libfuzzer ; make )
+          cp -r libfuzzer/fuzz.exe libfuzzer/seeds artifacts/linux-libfuzzer
+
+          mkdir -p artifacts/linux-trivial-crash
+          (cd trivial-crash ; make )
+          cp -r trivial-crash/fuzz.exe trivial-crash/seeds artifacts/linux-trivial-crash
+
+          mkdir -p artifacts/linux-trivial-crash-asan
+          (cd trivial-crash ; make clean; make CFLAGS='-fsanitize=address -fno-omit-frame-pointer')
+          cp -r trivial-crash/fuzz.exe trivial-crash/seeds artifacts/linux-trivial-crash-asan
+
+          mkdir -p artifacts/linux-libfuzzer-rust
+          (cd libfuzzer-rust ; make )
+          cp -r libfuzzer-rust/fuzz/target/x86_64-unknown-linux-gnu/release/fuzz_target_1 artifacts/linux-libfuzzer-rust
+
+          # AFL v2.75b
+          mkdir -p artifacts/linux-trivial-crash-afl
+          git clone https://github.com/google/AFL
+          (cd AFL; git checkout 82b5e359463238d790cadbe2dd494d6a4928bff3; make afl-gcc afl-fuzz afl-as)
+          export AFL_CC_PATH=$PWD/AFL/afl-clang
+          (cd trivial-crash ; make clean; make CC=$AFL_CC_PATH)
+          cp -r trivial-crash/fuzz.exe trivial-crash/seeds artifacts/linux-trivial-crash-afl
+    - uses: actions/upload-artifact@v2.1.4
+      with:
+        name: integration-test-artifacts
+        path: src/integration-tests/artifacts
+  build-integration-tests-windows:
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v2
+    - run: |
+          Set-ExecutionPolicy Bypass -Scope Process -Force 
+          $ProgressPreference = 'SilentlyContinue'
+          Invoke-Expression (Invoke-RestMethod 'https://chocolatey.org/install.ps1')
+          choco install llvm
+          choco install make
+          $env:Path += ";C:\Program Files\LLVM\bin;C:\ProgramData\chocolatey\bin"
+          
+          
+          cd src/integration-tests
+          
+          mkdir artifacts/windows-libfuzzer
+          cd libfuzzer
```

Comment:
FYI, PS has [`Push-Location`](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/push-location?view=powershell-7.1) and `Pop-Location`, which I believe get `pushd` and `popd` aliases by default.

Reply:
I have more tests to add in the future.  I'll try that out then and rework this.